src/sdk/formatting/formatCurrency.ts

```diff
@@ -0,0 +1,9 @@
+import { formatNumber } from './formatNumber';
+
+export const formatCurrency = (number: number, locale: string = 'en', currency: string = 'USD', wholeUnitsOnly: boolean = false): string =>
```

Comment:
We should follow convention and add `CurrencyFormatOptions = { locale?: string, currency?: string }`